compile/WINDOWS_INSTRUCTIONS.md

```diff
@@ -0,0 +1,59 @@
+### Steps to compile IRBEM x64 on Windows
+1. Download and install MSYS2 https://www.msys2.org/
+2. Following the installation steps and update the packages:
+```
+pacman -Syu
+```
+After packages are updated, the terminal will be closed.
```

Comment:
Will be closed? It doesn't close automatically. If this is asking the user to close the terminal it should say so.

Reply:
It does not do that now. But the previous version of MSYS did closed the terminal during the installation.